events/wc_druidic_events.txt

```diff
@@ -111,7 +129,125 @@ character_event = {
 		73 = { province_event = { id = WCDRU.340 days = 365 } }
 	}
 }
-# Starts story
+### You got war troubles, help is needed, worgen seem legit - druid version
```

Comment:
Don't duplicate events. First, one event chain slows down the game 2 times less. Second... it's duplication, so when one event chain is edited, you must change another one. I recommend you to use custom localization.
https://ck2.paradoxwikis.com/Localisation#Custom_commands

Reply:
The event chains merge after only 2 events; the first determines the artifact used, with the scourge factor being specific to the book. The second determines the mage_target as a mage or druid, with different trait requirements involved (or generated if it needs a new courtier).

There was a third event, but since it was an outright duplicate I've deleted it. I used the info in the link to make the DESC change based on which artifact is involved. If artifacts aren't enabled then it should randomly select one of the 2 descriptions. 

Determining the description based on the preceding event that triggered it would have been ideal, but I wasn't sure how and this should achieve the same effect.

Reply:
Both problems are still there. There's 2 start events, they slow down the game, not the event chains. I don't see any reason why there should be 2 start events. Conditions can be easily merged without problems. Also, why the Scourge factor is specific to the book? It's strange.

Reply:
Done; event WCDRU.298 had been deleted and the conditions merged. The Scourge factor was so that neighboring realms would have a chance of firing the event without needing to be at war with them, since the wars tend to be a quick curbstomp by Scourge doomstacks. 

Lore-wise, Dalaran takes an immediate interest in the plague outbreak, and takes measures against the Scourge long before the city itself is directly threatened. Likewise, the king of Gilneas has Arugal summon the Worgen while they are still safe behind the Greymane Wall.

If you were asking why the book and not the scythe had the Scourge factor, that was arbitrary and has been changed.